.circleci/scripts/release-bump-manifest-version

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+if [[ "${CI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CI environment variable must be set to true'
+    exit 1
+fi
+
+if [[ "${CIRCLECI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CIRCLECI environment variable must be set to true'
+    exit 1
+fi
+
+printf '%s\n' 'Updating the manifest version if needed'
+
+version="${CIRCLE_BRANCH/Version-v/}"
+cat <<< "$(jq ".version = \"$version\"" app/manifest.json)" > app/manifest.json
```

Comment:
Shellcheck shows a warning for this line (https://github.com/koalaman/shellcheck/wiki/SC2094)

Generally speaking, reading and writing to the same file in a pipeline seems like a bad idea. But I'm having difficulty imaging what could go wrong in this specific case.

Reply:
Yeah I believe this is safe since the inline heredoc is buffered but I'll split it into multiple lines to be explicit